lib/statsd_middleware.rb

```diff
@@ -31,16 +77,24 @@ def call(env)
     if path_parameters
       controller = path_parameters[:controller]
       action = path_parameters[:action]
+      source_app = env['HTTP_SOURCE_APP_NAME']
```

Comment:
```ruby
    # We should never use a dynamic path to apply the tag for the instrumentation,
    # since this will permit a rogue actor to increase the number of time series
    # exported from the process and causes instability in the metrics system.
```

@wyattwalter do you see a problem with this given the ominous comment ☝️ . We do perform a whitelist later on.

Reply:
I suppose my real question is: does this warrant some changes in our prometheus storage config?

I came across [a prior postmortem](https://github.com/department-of-veterans-affairs/vets.gov-team/blob/master/Postmortems/2017-02-21-prometheus-metrics-count.md), where we created a new label that had an infinite number of possible values. In practice, I'm not sure where that breaking point is of how many possible values warrant config changes.

Is trying to calculate the storage impact upfront a practical solution? Or is it best to merge and then monitor?